repo/migrations/Migration030.go

```diff
@@ -0,0 +1,243 @@
+package migrations
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+)
+
+const (
+	am01EthereumRegistryAddressMainnet = "0x5c69ccf91eab4ef80d9929b3c1b4d5bc03eb0981"
+	am01EthereumRegistryAddressRinkeby = "0x5cEF053c7b383f430FC4F4e1ea2F7D31d8e2D16C"
+	am01EthereumRegistryAddressRopsten = "0x403d907982474cdd51687b09a8968346159378f3"
+	am01UpVersion                      = 31
+	am01DownVersion                    = 30
+)
+
+// am01 - required migration struct
+type am01 struct{}
+
+type Migration030 struct {
+	am01
+}
+
+// Up - upgrade the state
+func (am01) Up(repoPath, dbPassword string, testnet bool) error {
```

Comment:
Since `APIPool` is being added in this migration should the regular `API` fields be removed?